lstchain/tools/lstchain_create_cat_B_calibration_file.py

```diff
@@ -123,7 +124,7 @@ def setup(self):
 
         group_name = 'tel_' + str(tel_id)
 
-        self.log.debug(f"Open output file {self.output_file}")
+        self.log.info(f"Open output file {self.output_file}")
```

Comment:
Maybe a good opportunity to use the lazy formatting in logging instead of fstrings

Reply:
Actually, I prefer to keep a correspondence between the two calibration scripts (Cat-A and Cat-B), these messages are mainly one per job, do not imply calculations and they are in the main function, is this lazy stuff so important here ? (Also I find more readable this format ;-) )